### DISCUSSION TITLE
深谈个人对新技术的看法 - 愧怍的小站

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#212] [General]
# 深谈个人对新技术的看法 - 愧怍的小站

目前技术圈的发展速度可谓是有目共睹，尤其是前端，每隔一段时间就出新的技术，可以说让很多初学者非常畏惧，没有一个明确的方向不知道从何学起。

https://kuizuo.cn/talk-new-technologies-opinion/#comments

**Comment by NOeSIS-yu** [comment]
不错，但学习真的很消耗时间，很多时候时间都花在钻牛角尖上面去了，我也觉得直接拿一个demo来实践是最好的学习方式，单独学写代码，就跟背死书一样，很容易让人感到烦躁和无趣，但如果能做出东西来就有正面的激励作用

**Reply by kuizuo** [reply]
不要让自己失去学习的兴致就是最好的学习方式！
